bsi/src/main/java/org/roaringbitmap/bsi/longlong/Roaring64NavigableMapSliceIndex.java

```diff
@@ -0,0 +1,622 @@
+package org.roaringbitmap.bsi.longlong;
+
+import org.roaringbitmap.bsi.BitmapSliceIndex;
+import org.roaringbitmap.bsi.Pair;
+import org.roaringbitmap.longlong.Roaring64NavigableMap;
+
+import java.io.*;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.IntStream;
+
+/*
+ * a Roaring64NavigableMap based bsi implementation
+ * The serialization format is compatible with bsi implementation of roaring(Go)
```

Comment:
Could you provide some link to the Go repository?

Reply:
ok